### ISSUE TITLE
Duplicate implementation of MMEHashProvider

### ISSUE THREAD

**Issue by frederoni** [mapbox/mapbox-events-ios#70]
When integrating the iOS Maps SDK and mapbox-events-ios, we see the following output:
```
Class MMEHashProvider is implemented in both 
…/Frameworks/Mapbox.framework/Mapbox (0x106a6d288) 
and 
…/MapboxMobileEvents.framework/MapboxMobileEvents (0x1061a6a58).
One of the two will be used. Which one is undefined.
```

It only triggers on MMEHashProvider but I would assume all implementations are duplicated.

cc @rclee

**Comment by tezqa** [comment]
Same issue here

**Comment by friedbunny** [comment]
This’ll be fixed in the `ios-v4.5.0-beta.1` release of the maps SDK.
